HCCGo/postinstall.js

```diff
@@ -1,12 +1,42 @@
 #!/usr/bin/env node
-var exec = require('child_process').exec;
+const spawn = require('child_process').spawnSync;
+const fs = require('fs');
+const target = "1.4.3";
 
-exec('./node_modules/.bin/electron-rebuild -w ffi',
-    function(error,stdout,stderr){
-    
-    });
+var rebuilder = "";
+var brace = "";
 
-exec('./node_modules/.bin/electron-rebuild -w ref',
-    function(error,stdout,stderr){
-    
-    });
+if (process.platform == 'win32') {
+    rebuilder = "electron-rebuild.cmd";
+    brace = "\\";
+} else {
+    rebuilder = "electron-rebuild";
+    brace = "/";
+}
+
+// nslog fixer
+try {
+    fs.unlinkSync(__dirname + brace + 'node_modules'
+                  + brace + 'nslog' + brace + 'build' + brace 
+                  + 'Release' + brace + 'nslog.node');
+} catch (e) {
+    console.log("nslog.node doesn't exist\n");
+}
+
+const ref = spawn(__dirname + brace + 'node_modules' + brace + '.bin' 
```

Comment:
You should use the `Path` module, especially the [path.join](https://nodejs.org/api/path.html#path_path_join_paths) function